examples/ex_interacting_multiple_model_se3.py

```diff
@@ -0,0 +1,197 @@
+from pynav.filters import ExtendedKalmanFilter
+from pynav.lib.states import VectorState, SO3State, SE3State
+from pynav.datagen import DataGenerator
+from pynav.types import StampedValue, StateWithCovariance
+from pynav.utils import GaussianResult, GaussianResultList, MonteCarloResult
+from pynav.utils import randvec
+
+from pynav.utils import monte_carlo, plot_error
+
+from pynav.lib.models import (
+    BodyFrameVelocity,
+)
+from pynav.lib.models import RangePoseToAnchor
+from pylie import SO3, SE3
+
+import numpy as np
+from typing import List
+import time
+from matplotlib import pyplot as plt
+from pynav.imm import InteractingModelFilter, run_interacting_multiple_model_filter
+from pynav.imm import run_time_varying_Q_filter
+from pynav.imm import ImmResultList
+
+
+def make_range_models_iterated_ekf(R):
```

Comment:
any way to make these examples simpler, making them more like the other ones?

Examples should be a "quickstart"